components/autoComplete/autoComplete.js

```diff
@@ -284,18 +285,21 @@ class AutoComplete extends Component {
 
   render() {
     const {
+      className,
       dataAttrs = {},
       disabled,
       label,
       name,
       placeholder,
     } = this.props;
-    const mods = this.props.mods ? this.props.mods.slice() : [];
-
-    const otherProps = ejectOtherProps(this.props, AutoComplete.propTypes);
+    const mods = [];
 
     this.state.open && mods.push('open');
-    const className = getClassNamesWithMods('ui-autocomplete', mods);
+
+    const classNames = classnames(
+      getClassNamesWithMods('ui-autocomplete', mods),
+      className
+    );
```

Comment:
`const mods = getClassNamesWithMods('ui-autocomplete', { 'open': this.state.open });`

`const classNames = classnames(mods, className);`

I'm not sure will it be better or not this way, so it's up to you to decide =)